examples/pytorch/rgcn/README.md

```diff
@@ -5,42 +5,34 @@
 * Author's code for link prediction: [https://github.com/MichSchli/RelationPrediction](https://github.com/MichSchli/RelationPrediction)
 
 ### Dependencies
-* PyTorch 1.10
-* rdflib
-* pandas
-* tqdm
-* TorchMetrics
+- rdflib
+- torchmetrics
 
-```
-pip install rdflib pandas
+Install as follows:
+```bash
+pip install rdflib
+pip install torchmetrics
 ```
 
-Example code was tested with rdflib 4.2.2 and pandas 0.23.4
+How to run
+-------
```

Comment:
Are you using the same hyperparameters for all the datasets? Do you observe performance differences?

Reply:
Yes, same hyperparameters for all datasets. And yes, there are slight differences (< 5%), but not significant.

Reply:
Could you list the detailed numbers before and after the changes?

Reply:
I am using the same parameters as the previous default ones. In general, I will only change the default parameters when (1) they are different than the paper as well as (2) they can noticeably change(improve) the performance. 
I agree with you we should include the changed parameters in PR or just file an Issue first and then refer it.


I thought you meant if all datasets share a same set of parameters. Here, of course, as I removed all input arguments, all datasets use a same set of parameters (as the previous default ones). But in the paper, every dataset has their own tuned parameters, like num_basis, weigh decay etc.  Old implementation can be more approaching to paper's setting, by setting input argument. But, I was saying those different argument settings just have minor effect on results.

When we compare the results from this PR vs. the old implementation with default parameters, I see very consistent results.

Reply:
Thank you for your reminding though. I just found I changed a default setting "self_loop=false", in my yesterday's commit for `entity.py` file. The reason is, I think in the paper, the authors include the self loop msg for their full graph training; and a reference implementation of pyG rgcn example also has root(self) msg by default. But as I observed, w/wo self loop msg seems have minor impact on performance. So, I just reverted my change and had "self_loop=false" back, for `entity.py` file.

Reply:
I was suggesting adding a table in the PR description and list the detailed experiment metrics before and after this PR. As you said, you changed the README file so that the same hyperparameters are used for all datasets. I'm ok with this, but we want to have a record of the numbers in case we want to refer to them in the future.

Something like below

| file | dataset | before | after |
| --- | -------- |------ | ----- |
|entity.py | AIFB | ... | ... |

Reply:
Oh.. okay, I see. I will add such table in this PR description.